### PR TITLE
rename develop tools to develop menu

### PR DESCRIPTION
**Pull request by Tony---Zhang**
## Type of change
- [x] New feature (non-breaking change which adds functionality)

## In the scope
Rename `Develop Tools` to `Develop Menu`

## Checklist
1. [x] Makes your submission pass tests?
2. [x] Have you lint your code locally before submission?
3. [x] Have you updated the wiki of your change?

### CODE REVIEW

File: app/src/dev/java/com/thoughtworks/ark/developmenu/DevelopMenuActivity.kt [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
-class DevelopToolsActivity : AppCompatActivity() {
+class DevelopMenuActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setupWindow()
 
         setContent {
-            MaterialTheme {
-                DevelopToolsScreen()
+            APPTheme {
```
Comment (jie-meng) [inline app/src/dev/java/com/thoughtworks/ark/developmenu/DevelopMenuActivity.kt:29]: Suggest rename "APPTheme" to "AppTheme", which same as AppCompatActivity

Comment (yefeidd) [reply app/src/dev/java/com/thoughtworks/ark/developmenu/DevelopMenuActivity.kt:29]: I will change it on the theme pr

File: app/src/dev/java/com/thoughtworks/ark/developmenu/DevelopMenuActivity.kt [hunk 2]
```diff
@@ -38,14 +41,17 @@ class DevelopToolsActivity : AppCompatActivity() {
...
+            modifier = Modifier
+                .fillMaxSize()
+                .background(color = APPTheme.colors.background)
         ) {
+            val context = LocalContext.current
             Text(
                 modifier = Modifier.padding(start = 16.dp, top = 16.dp, bottom = 8.dp),
-                text = "DevelopTools",
-                color = Color.Black,
+                text = context.getString(R.string.develop_menu_label),
```
Comment (yefeidd) [inline app/src/dev/java/com/thoughtworks/ark/developmenu/DevelopMenuActivity.kt:53]: using stringResource(id = xxx) may more make sense.

Comment (yefeidd) [reply app/src/dev/java/com/thoughtworks/ark/developmenu/DevelopMenuActivity.kt:53]: If you use `stringResource`, we can remove the logic of how to get context.

**Review by yefeidd — APPROVED** [review]
LGTM

**Pull request merged**